tests/unit/YourCompany.spec.ts

```diff
@@ -137,7 +137,7 @@ describe('YourCompany in an Alteration', () => {
 
   it('displays the Name Request information when NR data changes', async () => {
     store.state.stateModel.nameRequest.nrNumber = 'NR1234567'
-    store.state.stateModel.nameRequest.legalType = 'BEN'
+    store.state.stateModel.nameRequest.legalType = 'CR'
```

Comment:
This is a "corporation" NR ? It doesn't seem right.

Re: https://github.com/bcgov/business-edit-ui/blob/master/src/enums/entityTypes.ts

Reply:
Oh wait, you're trying to get the error here?

Reply:
Yessir, I was creating a Business type conflict and verifying.
Just a small extra check.

Reply:
Does the test name need to be updated?

Reply:
I don't think so, the name is still accurate. Sort of a supplementary check. A type mismatch doesn't actually block anything at this point, so when we get to the filing aspect of things, it will be useful to expand the testing there. 
I think*

Reply:
OK, your explanation makes sense. How about a code comment to explain it?

Reply:
Added